drm-ffi/drm-sys/src/platforms/linux/x86_64/bindings.rs

```diff
@@ -856,15 +856,6 @@ pub struct drm_syncobj_timeline_wait {
 }
 #[repr(C)]
 #[derive(Debug, Default, Copy, Clone, Hash, PartialEq, Eq)]
-pub struct drm_syncobj_eventfd {
```

Comment:
:eyes:

Reply:
CI runs on older headers it seems... *sigh*

Reply:
I think in the past the stance on this has been, if you need newer headers, just use the `bindgen`-feature. I tried to find an up-to-date PPA for libdrm on ubuntu and didn't find anyway. I also don't want to change CI to download and install libdrm manually. So the generated bindings will continue being best-effort in the interest of maintainability. I am sure this will trickle down to ubuntu 22.04 eventually.

Reply:
Yeah it's good to have somewhat older but consistent bindings, would be great if too-new bindings didn't end up here via irrelevant PRs in the first place.